benito-backend/src/main/kotlin/com/github/proyeception/benito/client/MedusaClient.kt

```diff
@@ -2,15 +2,31 @@ package com.github.proyeception.benito.client
 
 import com.fasterxml.jackson.core.type.TypeReference
 import com.github.proyeception.benito.connector.Connector
+import com.github.proyeception.benito.dto.MedusaFilter
 import com.github.proyeception.benito.dto.MedusaProjectDTO
+import com.github.proyeception.benito.dto.OrderDTO
 import com.github.proyeception.benito.exception.FailedDependencyException
 import org.slf4j.LoggerFactory
 
 open class MedusaClient(
-    private val medusaConnector: Connector
+        private val medusaConnector: Connector
 ) {
-    open fun projects(): List<MedusaProjectDTO> {
-        val response = medusaConnector.get("/projects")
+    open fun getProjects(
+            orderBy: OrderDTO? = null,
+            from: String? = null,
+            to: String? = null,
+            nameContains: String? = null,
+            tags: String? = null
+    ): List<MedusaProjectDTO> {
+        val endpoint = "/projects?"
+                .appendOrder(orderBy)
+                .appendParam("creation_date", from, MedusaFilter.GREATER_OR_EQUAL)
+                .appendParam("creation_date", to, MedusaFilter.LESS_OR_EQUAL)
+                .appendParam("title", nameContains.replaceWhitespaces(), MedusaFilter.CONTAINS)
+                .appendParam("tags", tags, MedusaFilter.IN)
```

Comment:
Ojo que el `in` chequea si el valor está dentro de una lista de valores y acá le estás mandando un único elemento. Dejémoslo simple ahora con un `eq` y cuando veamos cómo encaramos las categorías lo arreglamos.

Reply:
Sip, in era por si era una lista pero bueno vamos por el eq.
Modificado y probado, funciona pero me hace ruido que en la response no me está devolviendo lo que espero el back (me trae lista vacía) pero asumo que es un problema que tiene al tratar de convertir lo que devuelve strapi a una lista de string.

Más adelante también tendríamos que también como dijiste preguntar al back las categorías posibles para armar un combobox o algo